src/posts/2020-10-14-wcag2.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Introduction to WCAG 2
+description: WCAG is a set of accessibility guidelines published by WAI...
+category: Background
+author: Hamsa Harcourt
+further_reading:
+  - url: https://www.w3.org/TR/WCAG20/#intro-layers-guidance 
+    title:  Web Content Accessibility Guidelines (WCAG) 2.0
+    source: W3C 
+  - url:  https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines
+    title: Web Content Accessibility Guidelines
+    source: Wikipedia
+date: 2020-10-14
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+The **[Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (WCAG) are part of a series of web accessibility guidelines published by the **[Web Accessibility Initiative](https://www.w3.org/WAI/about/)** (WAI) . The WCAGs goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. 
+
+## WCAG 2.0
+
+WCAG 2.0 was published on 11 december 2008 and consists of 12 to 13 guidelines. It is organized around four principles which form the acronym `POUR`.  They include:
+
+1. Content must be `perceivable (P)`.
+
+- Provide a text alternative for all non-text content.
+- Provide synchronized alternatives for multimedia (such as captioned video, audio descriptions and so forth).
+- Information and structure must be separate from presentation.
+- Make it easy to distinguish foreground information from the background (for example good color contrast).
+
+2. Interfaces must be `operable (O)`. 
+
+- All functionality must be operable via keyboard.
+- All users must be able to control limits on their reading or interaction.
+- Users must be to avoid content that can cause seizures due to photosensitivity.
+- Provide mechanisms for users to find content, orientate themselves and navigate through it.
+- Help users avoid  mistakes and make it easier to correct mistakes when they occur.
+
+3. Content and controls must be `understandable (U)`.
+
+- Make text content readable and understandable.
+- Make the placement and functionality of content predictable.
+
+4. Content must be `robust (R)`.
+
+- Support compatibility with current and future user agents.
+- Ensure that content is accessible, or provide accessible alternatives.
+
+### Layers of guidance
+
+WCAG 2.0 provides various ways to help authors meet its standard called `layers of guidance`. They include the following :
```

Comment:
```suggestion
WCAG 2.0 provides various ways to help authors meet its standard called "layers of guidance". They include the following :
```

Quoting this as well. Can we also wrap "layers of guidance" to a link to the relevant page?

Reply:
Yes we can.